providers/SCConfigurationProvider.js

```diff
@@ -2,7 +2,17 @@
 
     'use strict';
 
-    angular.module('soundcloudify.core').provider('SCConfiguration', SCConfigurationProvider);
+    try{
+        angular.module('soundcloudify.core')
+            .provider('SCConfiguration', SCConfigurationProvider);
+    }
+    catch (err){ /*ignore*/ }
+
+    try{
+        angular.module('soundcloudify.background')
+            .provider('SCConfiguration', SCConfigurationProvider);
+    }
```

Comment:
Well, sorry that I didn't replied you earlier via email. I think this try catch approach isn't the way angular module is designed for.

These files located in `core` should always be in `core`. If you want to use the service in the background app, you just need to inject the `soundcloudify.core` module into the background app.

Please refer to this comment: https://github.com/ptgamr/upnext/pull/29/files#diff-2d77636971b1859b946f98b997c87b3cR22